internal/service/ipam/model_ipamsvc_ha_group_heartbeats.go

```diff
@@ -57,11 +57,7 @@ func (m *IpamsvcHAGroupHeartbeatsModel) Expand(ctx context.Context, diags *diag.
 	if m == nil {
 		return nil
 	}
-	to := &ipam.HAGroupHeartbeats{
-		Peer:                  flex.ExpandStringPointer(m.Peer),
-		SuccessfulHeartbeat:   flex.ExpandStringPointer(m.SuccessfulHeartbeat),
-		SuccessfulHeartbeatV6: flex.ExpandStringPointer(m.SuccessfulHeartbeatV6),
-	}
+	to := &ipam.HAGroupHeartbeats{}
```

Comment:
can't you remove the whole expand function itself since nothing is being set for this?

Reply:
Done . 
Removed a few other values too which were computed but added to Expand.